roomserver/api/output.go

```diff
@@ -195,3 +200,11 @@ type OutputRedactedEvent struct {
 	// The value of `unsigned.redacted_because` - the redaction event itself
 	RedactedBecause gomatrixserverlib.HeaderedEvent
 }
+
+// An OutputNewPeek is written whenever a user starts peeking into a room
+// using a given device.
+type OutputNewPeek struct {
```

Comment:
Random Q: Is it desirable for members in the room to know who is peeking?

Reply:
good question. in msc1776 you got this for free; in msc2753 it's kept private. in the end, privacy is probably best, otherwise it's effectively a tracking pixel.